internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -356,11 +361,16 @@ func (e *EnvStackConfig) cdnConfig() *template.CDNConfig {
 	return nil // no-op - return &template.CDNConfig{} when feature is ready
 }
 
-func (e *EnvStackConfig) publicHTTPConfig() template.HTTPConfig {
+func (e *EnvStackConfig) publicHTTPConfig() (template.HTTPConfig, error) {
+	ELBAccessLogsConfig, err := convertELBAccessLogsConfig(e.in.Mft)
```

Comment:
Typically exported variable names start with a capital letter, so in this scenario it would be like this :)
```suggestion
	elbAccessLogsConfig, err := convertELBAccessLogsConfig(e.in.Mft)
```

Reply:
What? 🥺 I thought I renamed it in small laters in my nits fix. I will change it now.